pwa/src/components/translations/translationTable.tsx

```diff
@@ -49,12 +51,18 @@ export default function TranslationTable({ tableName }) {
   };
 
   const handleDeleteTranslation = (id): void => {
-    if (confirm(`Do you want to delete this translation? With id ${id}`)) {
+    if (translations.length === 1 ? confirm(`Do you want to delete this translation? With id ${id}. If you delete this translation you wil also delete the Translation Table.`) 
+    :  confirm(`Do you want to delete this translation? With id ${id}`)) {
+      setLoadingOverlay(true);
       API.Translation.delete(id)
         .then(() => {
           setAlert({ message: `Deleted translation with id: ${id}`, type: "success" });
+          translations.length === 1 && navigate("/translation-tables")// removed the last translation table
           getTranslations();
         })
+        .finally(() => {
+          setLoadingOverlay(false);
+        })
```

Comment:
I'd put this underneath the `.catch()` for better readability. The `finally()` is last to be called and therefore I would say logically underneath everything.